ocs_ci/helpers/dr_helpers.py

```
@@ -3074,3 +3074,30 @@ def validate_protection_label(kind, namespace, protection_name=None):
+    )
+    gitops_private_repo_secret["metadata"][
+        "name"
+    ] = constants.GITOPS_PRIVATE_REPO_SECRET
+
+    gitops_private_repo_secret["stringData"]["url"] = config.ENV_DATA.get(
+        "dr_workload_repo_url"
+    )
+    gitops_private_repo_secret["stringData"]["password"] = config.clusters[
+        get_active_acm_index()
+    ].AUTH["github_ibm_odf_qe_ocs_workloads"]["gh_token"]
+    gitops_private_repo_secret_yaml = tempfile.NamedTemporaryFile(
```

Ruff (0.15.14)

[warning] Use a context manager for opening files

(SIM115)